client/components/drop-zone/test/index.jsx

```diff
@@ -37,7 +24,9 @@ describe( 'DropZone', function() {
 	} );
 
 	after( function() {
-		delete window.MutationObserver;
+		if ( global.window && global.window.MutationObserver ) {
```

Comment:
It was needed in the original test, but I agree, with `use-fake-dom` performing the [cleanup](https://github.com/Automattic/wp-calypso/blob/632678f9cb16a0e2a111ad65dc16da861df2dd60/client/lib/react-test-env-setup/index.js#L39-L47) in [`after`](https://github.com/Automattic/wp-calypso/blob/master/test/test/helpers/use-fake-dom/index.js), this shouldn't be necessary.